Task_Position_the_Cubes/index.css

```diff
@@ -1 +1,54 @@
-/*  Start coding below for POSITION THE CUBES TASK*/
+/*  Start coding below for POSITION THE CUBES TASK*/
+.container{
+    height: 100vh;
+    width: 100vw;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: black;
+}
+.first_cube{
+    height: 300px;
+    width: 300px;
+    background: rgb(103, 91, 198);
+    position: relative;
+    border: 1px solid white;
+}
+.second_cube{
+    height: 100px;
+    width: 100px;

```

Comment:
There should be properties defined for the background and border for the second_cube class.